src/styles/header.scss

```
@@ -38,6 +38,8 @@
       text-transform: uppercase;
       text-decoration: none;
       color: black;
+
+      transition: 1s;
       &:hover {
         color: $hover-color;
       }
```

```suggestion
      transition: color 1s;
      &:hover {
        color: $hover-color;
      }
```
Укажи св-во к которому применяется **transition**.